src/main/java/io/kestra/plugin/aws/eventbridge/PutEvents.java

```diff
@@ -0,0 +1,157 @@
+import java.util.List;
+
+@SuperBuilder
+@ToString
+@EqualsAndHashCode
+@Getter
+@NoArgsConstructor
+@Plugin(
+    examples = {
+        @Example(
+            title = "Send multiple custom events as maps to Amazon EventBridge so that they can be matched to rules",
+            code = {
+                "entries:",
+                "  - eventBusName: \"events\"",
+                "    source: \"Kestra\"",
+                "    detailType: \"my_object\"",
+                "    detail:",
+                "      message: \"hello from EventBridge and Kestra\"",
+            }
+        ),
+        @Example(
+            title = "Send multiple custom events as a JSON string to Amazon EventBridge so that they can be matched to rules.",
+            code = {
+                "entries:",
+                "  - eventBusName: \"events\"",
+                "    source: \"Kestra\"",
+                "    detailType: \"my_object\"",
+                "    detail: \"{\\\"message\\\": \\\"hello from EventBridge and Kestra\\\"}\"",
+                "    resources:",
+                "      - \"arn:aws:iam::123456789012:user/johndoe\"",
+            }
+        )
+    }
+)
+@Schema(
+    title = "Send multiple custom events to Amazon EventBridge so that they can be matched to rules."
+)
+public class PutEvents extends AbstractConnection implements RunnableTask<PutEvents.Output> {
+    @PluginProperty(dynamic = true)
+    @NotNull
+    @Schema(
+        title = "List of event entries to send to EventBridge",
+        description = "A list of at least one EventBridge entry."
+    )
+    private List<Entry> entries;
+
+    @Override
+    public PutEvents.Output run(RunContext runContext) throws Exception {
+        final long start = System.nanoTime();
+
+        try (var eventBridgeClient = client(runContext)) {
+            List<PutEventsRequestEntry> requestEntries = new ArrayList<>(entries.size());
+
+            for (Entry entry : entries) {
+                requestEntries.add(entry.toRequestEntry(runContext));
+            }
+
+            PutEventsRequest eventsRequest = PutEventsRequest.builder()
+                .entries(requestEntries)
+                .build();
+
+            PutEventsResponse putEventsResponse = eventBridgeClient.putEvents(eventsRequest);
+
+            runContext.metric(Timer.of("duration", Duration.ofNanos(System.nanoTime() - start)));
+            runContext.metric(Counter.of("failedEntryCount", putEventsResponse.failedEntryCount()));
+            runContext.metric(Counter.of("successfulEntryCount", entries.size() - putEventsResponse.failedEntryCount()));
+            runContext.metric(Counter.of("entryCount", entries.size()));
+
+            List<OutputEntry> outputEntries = new ArrayList<>(putEventsResponse.entries().size());
+            putEventsResponse.entries().forEach(
+                entry -> outputEntries.add(OutputEntry.builder()
+                    .eventId(entry.eventId())
+                    .errorCode(entry.errorCode())
```

Comment:
I have a the hard feeling that error lead to SUCCESS tasks, may I'm wrong ?

Reply:
The api only throws on certains error. But sending event failing will reult on a entry here with errormessage and error code. So yes that mean the kestra task would still be success. Do you think I should fail the task ? Even if only one event of x was fail ?

Reply:
IMHO it's ok to follow the same pattern as the aws api on this and allows to check for failure don't you think ?

Reply:
if the task bulk-sends multiple events, the ideal use case would be that if one of them fails to be sent, we make it easy for the user to troubleshoot and potentially manually redo the operation 

e.g. if only 2/100 events were misconfigured and couldn't be sent, it would be great to return them to users (maybe via an internal storage file) and mark the Execution as Failed

I think if any event fails to get sent, the execution should be marked as failed (usually, you only monitor/check for failed flows, you don't manually inspect Successful ones) so failing the execution is important

Reply:
but if I throw and mark the execution as failed I cannot produce an output to show which event failed right ?